src/selectors/nodes.js

```diff
@@ -232,3 +235,29 @@ export const getVisibleNodes = createSelector(
         }))
       : []
 );
+
+export const getNodeswithInputParams = createSelector(
+  [
+    getGraphNodes,
+    getEdgeIDs,
+    getNodeType,
+    getNodeTypeDisabled,
+    getEdgeSources,
+    getEdgeTargets,
+  ],
+  (nodes, edgeIDs, nodeType, nodeTypeDisabled, edgeSources, edgeTargets) => {
+    const nodesList = {};
+    for (const edgeID of edgeIDs) {
+      const source = edgeSources[edgeID];
+      const target = edgeTargets[edgeID];
+      if (
+        nodeType[source] === 'parameters' &&
+        nodeType[target] === 'task' &&
+        nodeTypeDisabled.parameters
```

Comment:
I think in terms of this selector being general, probably want to remove the `nodeTypeDisabled.parameters` check so this always works?